src/App.js

```diff
@@ -1,21 +1,17 @@
-import DetailPage from './DetailPage';
-import HomePage from './HomePage';
-import React, { useState } from 'react';
-import dotenv from "dotenv"
-dotenv.config()
+import DetailPage from "./DetailPage";
+import HomePage from "./HomePage";
+import React, { useState } from "react";
 
-const URL = process.env.API
+const URL = process.env.API;
```

Comment:
define this in constants.js file. benefit would be if you have to add authentication headers or other headers, there will be one source of change instead of changing it everywhere

Reply:
done